actor-tests/src/test/java/org/apache/pekko/dispatch/JavaFutureTests.java

```diff
@@ -40,25 +40,31 @@ public class JavaFutureTests extends JUnitSuite {
 
   @ClassRule
   public static PekkoJUnitActorSystemResource actorSystemResource =
-    new PekkoJUnitActorSystemResource("JavaFutureTests", PekkoSpec.testConf());
+      new PekkoJUnitActorSystemResource("JavaFutureTests", PekkoSpec.testConf());
 
   private final ActorSystem system = actorSystemResource.getSystem();
   private final Duration timeout = Duration.create(5, TimeUnit.SECONDS);
 
   @Test
   public void mustBeAbleToMapAFuture() throws Exception {
 
-    Future<String> f1 = Futures.future(new Callable<String>() {
-      public String call() {
-        return "Hello";
-      }
-    }, system.dispatcher());
-
-    Future<String> f2 = f1.map(new Mapper<String, String>() {
-      public String apply(String s) {
-        return s + " World";
-      }
-    }, system.dispatcher());
+    Future<String> f1 =
+        Futures.future(
+            new Callable<String>() {
+              public String call() {
+                return "Hello";
+              }
+            },
+            system.dispatcher());
```

Comment:
I think the format is ok. It is just more spread out - I would actually marginally prefer the new one.

The key thing is that we get the format to match what is defined in the format config instead of having some files that have different formats.

Reply:
You can test it with sbt javafmtAll .I have to revert everytime.

Reply:
I will merge, this thanks everyone.